lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if @price > 500000
+      true
+    else
+      false
+    end
```

Comment:
This is great.

It could also be written in a single line as:

`@price > 500000`, since that statement in and of itself returns a boolean value